### ISSUE TITLE
Remove phonegap specific files

### ISSUE THREAD

**Issue by kmeesters** [FirstLegoLeague/fllscoring#218]
Since we have not worked on it anymore and are not aiming to provide native apps for now, i'd suggest to remove phonegap/cordova specific files, (like the res folder) for easier maintainability

**Comment by rikkertkoppes** [comment]
We may also reconsider the goal itself. There are some new techniques like react native and nativescript as well if we were to pursue the idea of native apps.

**Comment by kmeesters** [comment]
Agreed, but in any case I'd rather see these as separate developments which can connect over Mhub, for example by other volunteers. In any case, remove the phonegap stuff?
